Dockerfile

```diff
@@ -57,6 +57,7 @@ ARG	BUILD_TARGET
 RUN	apt update && apt install -yqq wget
 RUN	wget -O - https://deb.nodesource.com/setup_18.x | bash
 RUN	apt update && apt install -yqq \
+	curl \
```

Comment:
Not technically necessary. But I've seen at least three people do a cheeky little `apt install curl` to put it in there for testing. So let's just put it in properly.